common/src/main/java/com/wynntils/features/internal/ItemStackTransformerFeature.java

```diff
@@ -75,6 +76,12 @@ protected void onInit(ImmutableList.Builder<Condition> conditions) {
         registerProperty(WynnItemMatchers::isIngredient, IngredientProperty::new);
     }
 
+    @Override
+    protected boolean onEnable() {
+        return (WebManager.isItemGuessesLoaded() || WebManager.tryLoadItemGuesses())
+                && (WebManager.isItemListLoaded() || WebManager.tryLoadItemList());
+    }
```

Comment:
This is an internal feature, so this effectively permanently loads item guesses/list.

Reply:
Yeah, but we need this for GearItemStack.

Reply:
Is there any reason to not permanently load the item data?

Reply:
My original idea was that if a certain web data such as item guesses isn't used by any features, it isn't loaded. We can ditch that if necessary.

Reply:
I just think it is a hassle to load it everywhere we need to, although per feature reloading of required web data is nice. (Not that reloading everything in WebManager is so much stuff though...)

Reply:
It might be nice to just permaload commonly-used like item map, while more niche things can be on demand.

Reply:
Chances are that you won't disable every single item map using feature.